docs/source/get_involved/contributing.rst

```diff
@@ -291,13 +285,13 @@ the ``index.rst`` file.
 
 To build the documentation locally, you need to install a few extra
 dependencies listed in
-`docs/requirements.txt <https://github.com/alan-turing-institute/sktime/blob/main/docs/requirements.txt>`__.
+`pyproject.toml <https://github.com/alan-turing-institute/sktime/blob/main/pyproject.toml>`__.
 
-1. To install extra requirements from the root directory, run:
+1. To install extra dependencies from the root directory, run:
 
    .. code:: bash
 
-      pip install -r docs/requirements.txt
+      pip install .[docs]
```

Comment:
hm, should we cater for the user who has a dev install but not the doc dependencies?
Would this not possibly un- or reinstall the rest?

Reply:
why not? Contributing to OS thought docs is usually a good start for may people.